test/acceptance/ember-cli-update-test.js

```diff
@@ -128,6 +130,14 @@ describe(function() {
     });
   }
 
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  beforeEach(function() {
+    sinon.stub(install, 'getBlueprintNameOverride').resolves();
```

Comment:
Why do this? I don't think our acceptance tests should do any stubbing.

Reply:
Removed! `getBlueprintNameOverride` originally did not read from disk